package.json

```diff
@@ -57,17 +57,17 @@
     "babel-preset-react": "^6.11.1",
     "coveralls": "^3.0.0",
     "enzyme": "^3.2.0",
-    "enzyme-adapter-react-16": "^1.1.0",
+    "enzyme-adapter-react-16": "^1.6.0",
     "enzyme-to-json": "^3.2.2",
     "gh-pages": "^1.2.0",
     "jest": "^21.2.1",
-    "react": "^16.3.2",
-    "react-dom": "^16.3.2",
+    "react": "^16.5.2",
+    "react-dom": "^16.5.2",
     "react-test-renderer": "^16.2.0",
     "standard": "^10.0.2"
   },
   "peerDependencies": {
-    "@material-ui/core": "^1.0.0 || ^3.0.0",
+    "@material-ui/core": "^1.0.0 || ^3.1.0",
```

Comment:
Are there things in v3.1.0 that we rely on and prevent us from being compatible with 3.0.0? If so, which features?

Reply:
Yes, the OutlineInput component was introduced in [v3.1.0](https://github.com/mui-org/material-ui/releases/tag/v3.1.0)

Reply:
Meh… But hey, we're still in beta, so we can just make this a semi-breaking change. :tada: Upgrading from 3.0.0 to 3.1.0 is pretty smooth (no breaking changes), so that's fine.

Unfortunately, the old typography types will be removed in the future (that's why they are deprecated). There is no way to disable the warnings. We'll have to see which typographies Material-UI uses and migrate to them.